ROADMAP.md

```diff
@@ -0,0 +1,191 @@
+# Testground Roadmap
+
+## Context
+
+> Testground is a platform for testing, benchmarking, and simulating distributed and peer-to-peer systems at scale.
+> It's designed to be multi-lingual and runtime-agnostic, scaling gracefully from 2 to 10k instances when needed.
+
+Testground was used successfully at Protocol Labs
+
+- TODO: note on libp2p interop
+- TODO: note on filecooin
+- TODO: note on IPFS v0.5.0 (massive DHT and Bitswap improvements)
+- TODO: note on libp2p (gossipsub 1.1 security extensions)
+
+## Vision
+
+Testground As A Service embodies our long-term vision.
+
+- A single, scalable platform that one or more organizations can use,
+- The ability to track the impact of a change in terms of stability & performance across multiple projects,
+- The ability to experiment with large-scale networks and simplify the integration testing of libraries used across deep stacks.
+
+Products with similar ideas:
```

Comment:
Do we want to discuss why testground is unique here? Why others are not meeting the requirements?

Reply:
I'll answer through the doc instead of using a comment, it looks like this now:

---

Products with similar ideas, but specialized in different areas:

- database: [CockroachDB performance tracker](https://cockroachdb.github.io/pebble/?max=local),
- browser: [Webkit Performance Dashboard](https://perf.webkit.org/v3/)

---

Let me know if you think we should add more!